### ISSUE TITLE
[CLI] Empty line in console log when using flag chatdownload --silent

### ISSUE THREAD

**Issue by superbonaci** [lay295/TwitchDownloader#986]
### Checklist

- [X] I have checked the [issue page](https://github.com/lay295/TwitchDownloader/issues?q=is%3Aissue) for duplicates
- [X] I am running the latest version ([download here](https://github.com/lay295/TwitchDownloader/releases/latest))

### Edition

Command Line Interface

### Describe your issue here

Without the flag all log is right:

```
$ TwitchDownloaderCLI chatdownload --id '758707576' -o chat.json -E
TwitchDownloaderCLI 1.54.1 Copyright (c) lay295 and contributors
[STATUS] - Downloading 97%
[STATUS] - Downloading + Embedding Images
[STATUS] - Backfilling commenter info
[STATUS] - Writing output file
```

With `--silent` there's an empty line:

```
$ TwitchDownloaderCLI chatdownload --id '758707576' -o chat.json -E --silent

[STATUS] - Downloading + Embedding Images
[STATUS] - Backfilling commenter info
[STATUS] - Writing output file
```

Also `Downloading` max value is `97%`, never shows `100%`.

**Comment by superbonaci** [comment]
Looks like the flag `--silent` also implies `--banner false` in all modes:

https://github.com/lay295/TwitchDownloader/blob/1afcc3ce2166ab128cb6ca0463fae46acc2e59b5/TwitchDownloaderCLI/Program.cs#L76

**Comment by ScrubN** [comment]
- This should be resolved after #1010 and #1019
